lib/trader/trade-bot-app.ts

```diff
@@ -5,7 +5,10 @@ import { tradeService } from './trade-service';
 export const tradeBotApp = express();
 
 tradeBotApp.get('/start-selling', async function (req, res) {
-  res.send(tradeService.startSelling())
+  let { maxRating, amount } = req.query
+  maxRating = maxRating && parseInt(maxRating, 10)
+  amount = amount && parseInt(amount, 10)

```

Comment:
One more explanation for amount: 
To not get too many requests bans, there are some checks and slow down mechanisms in play. First one is in api-interceptors. You can make max X request per second to fut-api. Currently it's set to 1 but you can easly change it to 2-3 without issues.  Second check is in trade job. SellXPlayers job is working in batches. every minute it sells X amount of players. (see trade-jobs constructor, the parameter sent to super). Making this optional will probably end up with too many requests ban (it's like 6-10 hours ban, not sure how much exactly)

Reply:
Got it. But I think leave a option here will give a reasonable choice for the user who wants to sell players more quickly and also want to take the risk of ban. Maybe I should add a default amount here?